tests/validations/test_table_definitions_check.py

```diff
@@ -129,13 +105,6 @@ def test_table_definitions_check_table_changed():
         "tests/data/test_allcorrect_definition.json", current_definitions
     )
     assert len(diff) == 1
-    assert diff[0]["R8"] == {
-        "errorinfo": {
-            "errortype": "R8",
-            "validation": "Geopackage must conform to given JSON definitions",
-        },
-        "errors": [
-            "Difference: Item root['test_different_table'] added to dictionary.",
-            "Difference: Item root['test_allcorrect'] removed from dictionary.",
-        ],
-    }
+    assert (
+        "RQ8" in diff[0]
+    )  # todo: Overwegen of we dit zo willen testen -> data structuur van de foutmelidng veranderen -> William heeft hier een voorstel voor
```

Comment:
Liefst dit niet in de code opnemen. Graag even bespreken via PR comments, via het Jira ticket en/of slack.